### ISSUE TITLE
Add a `Game` target to `ue4 build Shipping`

### ISSUE THREAD

**Issue by unrealcloud** [adamrehn/ue4cli#40]
Following the docs on this page: https://docs.adamrehn.com/ue4cli/descriptor-commands/build#top, I am trying to execute
`ue4 build Shipping`, to build this build of [TargetType.Game](https://github.com/unrealcloud/ActionRPGDemo/blob/main/Source/ActionRPG.Target.cs) (notice that it has no target appended, e.g. `ActionRPGGame.Target.cs`)

However, `ue4 build Shipping`, as per the docs, also tries to build the Editor modules, but the Editor does not support `Shipping` configuration.

Basically, `ue4 build Shipping` always fails...

To workaround this, we could add the `Game` target, to be used as `ue4 build Shipping Game`, which will not build Editor modules.

Or, a cli option `-noEditor`, such as `ue4 build Shipping -noEditor`.

Note: the BuildRunCook already supports `-SkipCookingEditorContent` and `-noCompileEditor`, but passing them to `ue4 build Shipping -SkipCookingEditorContent -noCompileEditor` doesn't seem to fix it.
